test/unit/core_utils_spec.js

```diff
@@ -210,6 +227,37 @@ describe("core_utils", function () {
     });
   });
 
+  describe("recoverJsURL", function () {
+    it("should get valid URLs without `newWindow` property", function () {
+      const inputs = [
+        "window.open('https://test.local')",
+        "app.launchURL('https://test.local')",
+        "app.launchURL('https://test.local', false)",
+        "xfa.host.gotoURL('https://test.local')",
```

Comment:
I think that it would be nice if we *also* checked that a spurious `true` argument is ignored for those formats where it should be, e.g. something like this:
```suggestion
        "window.open('https://test.local')",
        "window.open('https://test.local', true)",
        "app.launchURL('https://test.local')",
        "app.launchURL('https://test.local', false)",
        "xfa.host.gotoURL('https://test.local')",
        "xfa.host.gotoURL('https://test.local', true)",
```